OpenBSD su treats -c before username as class, and don't requires --

See: https://github.com/processone/ejabberd/pull/4320

Index: ejabberdctl.template
--- ejabberdctl.template.orig
+++ ejabberdctl.template
@@ -126,7 +126,7 @@ set_dist_client()
 exec_cmd()
 {
     case $EXEC_CMD in
-        as_install_user) su -s /bin/sh -c 'exec "$0" "$@"' "$INSTALLUSER" -- "$@" ;;
+        as_install_user) su -s /bin/sh "$INSTALLUSER" -c 'exec "$0" "$@"' "$@" ;;
         as_current_user) "$@" ;;
     esac
 }
